lighthouse-core/scripts/esm-utils.js

```diff
@@ -5,13 +5,30 @@
  */
 
 import module from 'module';
+import url from 'url';
+import path from 'path';
 
 const require = module.createRequire(import.meta.url);
 
 /**
  * Commonjs equivalent of `require.resolve`.
  * @param {string} packageName
  */
-export function resolveModulePath(packageName) {
+function resolveModulePath(packageName) {
   return require.resolve(packageName);
 }
+
+/**
+ * @param {ImportMeta} importMeta
+ */
+function createCommonjsRefs(importMeta) {
```

Comment:
I can see the benefit of reducing churn in this change by providing this, but it does seem like we should be looking to the future, too. e.g. new files that need a `_dirname + '/path/to/something'`, should really be doing `new URL('/path/to/something', import.meta.url)` rather than encouraging further use of `__filename` and `_dirname`

Reply:
agreed-#13953